alacritty.yml

```diff
@@ -480,62 +481,118 @@
 # - `action`: Execute a predefined action
 #
 #   - ToggleViMode
+#       Enable or Disable Using Vi key bindings
 #   - SearchForward
+#       Start a forward buffer search
 #   - SearchBackward
+#       Start a backward buffer search
 #   - Copy
+#       Copy the selection to the clipboard
 #   - Paste
+#       Paste from the system clipboard
 #   - PasteSelection
+#       Paste from the selection buffer
 #   - IncreaseFontSize
+#       Increase font size
 #   - DecreaseFontSize
+#       Decrease font size
 #   - ResetFontSize
+#       Reset font size to config value
 #   - ScrollPageUp
+#       Scroll Exactly One Page Up
 #   - ScrollPageDown
+#       Scroll Exactly One Page Down
 #   - ScrollHalfPageUp
+#       Scroll a half page up
 #   - ScrollHalfPageDown
+#       Scroll a half page down
 #   - ScrollLineUp
+#       Scroll one line up
 #   - ScrollLineDown
+#       Scroll one line down
 #   - ScrollToTop
+#       Scroll all the way to the top
 #   - ScrollToBottom
+#       Scroll all the way to the bottom
 #   - ClearHistory
+#       Clear the display buffer(s) to remove history
 #   - Hide
+#       Hide the Alacritty window
 #   - Minimize
+#       Minimize the Alacritty window
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
+#       Toggle full screen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty
 #   - ClearLogNotice
+#       Clear warning and error notices
 #   - ClearSelection
+#       Clear active selection
```

Comment:
I think we can do better in a description here. Rather than reusing the word `clear`, by replacing it with `Remove the active selection` we should be able to contribute to the name itself and help understand what this actually does.

So people know this doesn't just clear the selection clipboard or something like that.

Reply:
Advice taken.